### PR TITLE
chore: Add sender details in unread message

### PR DESCRIPTION
**Pull request by muhsin-k**
- Show sender name and company name in the first unread message
- Reduced the width of the unread message holder

### Preview
**Before**

**Screenshot 2021-05-17 at 8 34 49 PM**

**After (Left position)**
Screenshot 2021-05-17 at 8 35 22 PM

**After (Right position)**

Screenshot 2021-05-17 at 8 36 13 PM

### CODE REVIEW

File: app/javascript/widget/views/Unread.vue [hunk 1]
```diff
@@ -86,6 +94,9 @@ export default {
 
       return '';
     },
+    showSender(index) {
```
Comment (pranavrajs) [inline app/javascript/widget/views/Unread.vue:97]: !index?

File: app/javascript/widget/views/Unread.vue [hunk 2]
```diff
@@ -11,13 +11,16 @@
       </button>
     </div>
     <div class="unread-messages">
-      <agent-bubble
-        v-for="message in unreadMessages"
+      <unread-message
+        v-for="(message, index) in unreadMessages"
         :key="message.id"
         :message-id="message.id"
+        :show-sender="showSender(index)"
```
Comment (pranavrajs) [inline app/javascript/widget/views/Unread.vue:18]: I guess we can write !index directly here.

File: app/javascript/widget/components/UnreadMessage.vue [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  </div>
+</template>
+
+<script>
+import messageFormatterMixin from 'shared/mixins/messageFormatterMixin';
+import Thumbnail from 'dashboard/components/widgets/Thumbnail';
+import configMixin from '../mixins/configMixin';
+import { isEmptyObject } from 'widget/helpers/utils';
+export default {
+  name: 'AgentMessageBubble',
```
Comment (pranavrajs) [inline app/javascript/widget/components/UnreadMessage.vue:25]: Please rename this to the file name.

**Review by pranavrajs — DISMISSED** [review]

File: app/javascript/widget/components/UnreadMessage.vue [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+      }`;
+    },
+    avatarUrl() {
+      // eslint-disable-next-line
+      const BotImage = require('dashboard/assets/images/chatwoot_bot.png');
+      const displayImage = this.useInboxAvatarForBot
+        ? this.inboxAvatarUrl
+        : BotImage;
+      return !isEmptyObject(this.sender)
+        ? this.sender.avatar_url
```
Comment (nithindavid) [inline app/javascript/widget/components/UnreadMessage.vue:55]: Please use camelCase here and for `available_name`

File: app/javascript/widget/components/UnreadMessage.vue [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+    },
+  },
+};
+</script>
+
+<style lang="scss">
+@import '~widget/assets/scss/variables.scss';
+.chat-bubble {
+  max-width: 85%;
+  padding: $font-size-medium;
```
Comment (nithindavid) [inline app/javascript/widget/components/UnreadMessage.vue:72]: use $space for padding

File: app/javascript/widget/views/Unread.vue [hunk 6]
```diff
@@ -64,6 +68,10 @@ export default {
     showCloseButton() {
       return this.unreadMessageCount && this.hideMessageBubble;
     },
+    sender() {
+      const [firstMessage] = this.unreadMessages;
+      return firstMessage.sender ? firstMessage.sender : {};
```
Comment (nithindavid) [inline app/javascript/widget/views/Unread.vue:73]:
```suggestion
      return firstMessage.sender || {};
```

**Review by nithindavid — CHANGES_REQUESTED** [review]

**Review by pranavrajs — APPROVED** [review]

**Pull request merged**